src/main/resources/bootstrap.yml

```diff
@@ -101,7 +101,7 @@ jackson:
 aws:
   cognito:
     region: eu-west-1
-    userpoolid: eu-west-1_64ShtC2tS
+    userpoolid: eu-west-1_jnPKAXCKm
```

Comment:
Why did you change this?

Reply:
I just checked: I use eu-west-1_WsTxYUHyC when testing from localhost (both on server and frontend of course)

Reply:
Earlier I was using the pool ends with eu-west-1~~~~~z4 which was created in dev. I did not see it yesterday, Maybe it is deleted. In order to test yesterday's changes, I had to point to some user poll. But I should not have committed this